.buildkite/pull_request_pipeline.yml

```diff
@@ -123,6 +123,24 @@ steps:
       manual:
         allowed: true
 
+  - label: "Observability SRE container smoke test"
+    key: "observability-sre-container-smoke-test"
+    agents:
+      provider: gcp
+      imageProject: elastic-images-prod
+      image: family/platform-ingest-logstash-ubuntu-2204
+      machineType: "n2-standard-4"
+      diskSizeGb: 64
+    retry:
+      automatic:
+        - limit: 3
+    command: |
+      set -euo pipefail
+      source .buildkite/scripts/common/vm-agent.sh
+      ./gradlew --stacktrace artifactDockerObservabilitySRE
+      docker run docker.elastic.co/logstash/logstash-observability-sre:8.19.0-SNAPSHOT \
```

Comment:
Is there a pattern for invoking containers after building them? Thinking of how we want to avoid hard coding version here.